p2p/discovery.py

```diff
@@ -42,7 +42,8 @@
 
 from eth_hash.auto import keccak
 
-from p2p.cancel_token import CancelToken
+from cancel_token import CancelToken
+
 from p2p.exceptions import NoEligibleNodes, OperationCancelled
```

Comment:
We need to get rid of `p2p.exceptions.OperationCancelled` and use the one from cancel_token instead

Reply:
https://github.com/ethereum/py-evm/pull/1113